docs/user_guide/assessment.rst

```diff
@@ -258,6 +258,223 @@ across groups and also the difference and ratio between the maximum and minimum:
+
+.. _demographic_parity:
+
+Demographic parity
+^^^^^^^^^^^^^^^^^^
+Demographic parity is a fairness metric whose goal is to ensure a machine 
+learning model's predictions are independent of membership in a sensitive 
+group. In other words, demographic parity is achieved when the probability 
+of a certain prediction is not dependent on sensitive group membership. In 
+the binary classification scenario, demographic parity refers to equal 
+selection rates across groups. For example, in the context of a resume 
+screening model, equal selection would mean that the proportion of 
+applicants selected for a job interview should be equal across groups.
+
+We mathematically define demographic parity using the following 
+set of equations.
+A classifier :math:`h` satisfies demographic parity under a distribution 
+over :math:`(X, A, Y)` if its prediction :math:`h(X)` is statistically
+independent of the sensitive feature :math:`A`. This is equivalent to
+:math:`\E[h(X) \given A=a] = \E[h(X)] \quad \forall a`. :footcite:`agarwal2018reductions`
+
+In the case of regression, a predictor :math:`f` satisfies demographic parity
+under a distribution over :math:`(X, A, Y)` if :math:`f(X)` is independent
+of the sensitive feature :math:`A`. This is equivalent to
+:math:`\P[f(X) \geq z \given A=a] = \P[f(X) \geq z] \quad \forall a, z`.
+:footcite:`agarwal2019fair` Another way to think of demographic parity in a 
+regression scenario is to compare the average predicted value across groups.
+Note that in the Fairlearn API, :func:`fairlearn.metrics.demographic_parity_difference` 
+is only defined for classification. 
+
+.. note::
+   Demographic parity is also sometimes referred to as *independence*, *group fairness*, *statistical parity*, and *disparate impact*.
+
+Failing to achieve demographic parity could generate allocation harms. 
+Allocation harms occur when AI systems allocate 
+opportunities, resources, or information differently across different 
+groups (for example, an AI hiring system that is more likely to advance resumes 
+of male applicants than resumes of female applicants regardless of qualification). 
+Demographic parity can be used to assess the extent of allocation harms because it 
+reflects an assumption that resources should be allocated proportionally 
+across groups. Of the metrics described in this section, it can be the easiest 
+to implement. However, operationalizing fairness using demographic parity 
+rests on a few assumptions: that either the dataset is not a good representation  
+of what the world actually looks like (e.g., a resume assessment system that is 
+more likely to filter out qualified female applicants due to an organizational 
+bias towards male applicants, regardless of skill level), or that the dataset 
+is an accurate representation of the phenomena being modeled, but the 
+phenomena itself is unjust. In reality, this may not be the case. The
```

Comment:
Can we add a parenthetical example to help readers understand what it might mean for "the phenomena itself [to be] unjust"? Maybe something about "predictive policing"? There may be language in the user guide sections on construct validity (specifically consequential validity) or abstraction traps (formalism or framing traps, specifically) that might be helpful here (or you may want to give a quick parenthetical example and then link to one of those sections)

Reply:
I added the following example in bold: 

> However, operationalizing fairness using demographic parity rests on a few assumptions: that either the dataset is not a good representation  of what the world actually looks like (e.g., a resume assessment system that is more likely to filter out qualified female applicants due to an organizational bias towards male applicants, regardless of skill level), or that the dataset is an accurate representation of the phenomena being modeled, but the phenomena itself is unjust **(e.g., consider the case of predictive policing, where a system created to predict crime rates may correctly predict higher crime rates for certain areas, but simultaneously fail to consider that those higher rates may be caused by disproportionate policing and overcrimininalization of those areas).** 

Is this what you're thinking of?

Reply:
Looks great! thanks!!